portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,29 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>About Ben Xu</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="wrapper">
+        <div id="nav">
+            <h2><a href="index.html">Ben Xu</a></h2>
+            <ul>
+                <li><b><a href="projects.html">projects</a></b></li>
+                <li><b><a href="https://benxu.notion.site/752cd9c416d743c4a91d8e199c9e964e?v=497a1ee197ac4f238ed7deaf01c8a7df">blog</a></b></li>
+                <li><b><a href="https://github.com/benxu3">github</a></b></li>
+                <li><b><a href=>contact</a></b></li>
+            </ul>
+        </div>
+        <div id="content">
+            <h2>About</h2>
+            <p>Hi nice to meet you! 👋 My name is Ben, I'm a student at the University of Toronto studying computer science, math, and statistics.</p> 
+            <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+            <p>Click here to get to know me a bit better:</p>
```

Comment:
Maybe the following would be clearer:

```suggestion
            <p>Click the button below to get to know me a bit better:</p>
```